cspell.yaml

```diff
@@ -2,15 +2,48 @@ version: '0.2'
 language: en
 flagWords:
   - teh
-ignorePaths:
-  - .github/CODEOWNERS*
-  - .github/policies/**
+ignorePaths:  
   - '**/examples/**'
   - '**/package-lock.json'
   - cspell.json
   - cspell.yaml
+  - node_modules/**
+
+  # Exclude all files in the repo which aren't in specification/. cspell does 
```

Comment:
Can we use glob pattern `*` to match all files in root without listing them individually?